newrelic/resource_newrelic_notifications_destination_test.go

```diff
@@ -0,0 +1,114 @@
+//go:build integration
+// +build integration
+
+package newrelic
+
+import (
+	"fmt"
+	"github.com/newrelic/newrelic-client-go/pkg/ai"
+	"github.com/newrelic/newrelic-client-go/pkg/notifications"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestNewRelicNotificationDestination_Basic(t *testing.T) {
+	resourceName := "newrelic_notification_destination.test_foo"
+	rand := acctest.RandString(5)
+	rName := fmt.Sprintf("tf-notifications-test-%s", rand)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccNewRelicNotificationDestinationDestroy,
+		Steps: []resource.TestStep{
+			// Test: Create
+			{
+				Config: testNewRelicNotificationDestinationConfigByType(rName, "webhook", `{
+					type = "BASIC"
+					user = "test-user"
+					password = "pass123"
+				}`, `{
+					key = "url"
+					value = "https://webhook.site/94193c01-4a81-4782-8f1b-554d5230395b"
+				}`),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNewRelicNotificationDestinationExists(resourceName),
+				),
```

Comment:
Can we add an `import` test step here?

Here's a quick [example](https://github.com/newrelic/terraform-provider-newrelic/blob/main/newrelic/resource_newrelic_entity_tags_test.go#L41-L46).

Reply:
Done 👍